server/controllers/users.js

```diff
@@ -32,7 +32,7 @@ export function getUserById(req, res, next) {
     id: req.params.id
   };
 
-  return usersService.getUserById(params)
+  return usersService.getUserById(params, UsersModel)
```

Comment:
It was supposed to be passing the model previously but wasn't. That pattern was established when Duy isolated the model, before the major restructuring.